src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -159,16 +161,26 @@ public void updatePlayer(@NotNull TabPlayer tabPlayer) {
     }
 
     public CompletableFuture<Component> getHeader(@NotNull TabPlayer player) {
-        return Placeholder.replace(plugin.getSettings().getHeader(
-                        plugin.getSettings().getServerGroup(player.getServerName())), plugin, player)
+        if (indexheader >= plugin.getSettings().getHeaderListSize(plugin.getSettings().getServerGroup(player.getServerName()))){
+            indexheader = 0;
+        }
+        CompletableFuture<Component> headercomponent = Placeholder.replace(plugin.getSettings().getHeader(
```

Comment:
Could you camelCase this variable name? (`headercomponent` -> `headerComponent`) -- same with `footercomponent` below.)